adhocracy-plus/assets/scss/components/_poll.scss

```diff
@@ -157,6 +157,7 @@ $checkbox-size: 20px;
 
     i {
         color: $brand-tertiary;
+        margin-left: -1.25 * $spacer;
```

Comment:
this gets a little bit cut off to the left on screens below 768px.
Oops, it got just merged. Just leaving this mention here then. @phillimorland

Reply:
ah cheers!

Reply:
Oh, sorry!